src/lib/components/Header/index.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview A responsive header that can trigger a side-nav.
+ */
+
+import {expandSideNav} from "../../actions";
+
+class Header extends HTMLElement {
+  connectedCallback() {
+    this.hamburgerBtn = this.querySelector(".web-header__hamburger-btn");
+    this.hamburgerBtn.addEventListener("click", expandSideNav);
+  }
+
+  manageFocus() {
```

Comment:
Maybe comment where this is called from.

Alternatively, is there any precedent to overloading `.focus()` itself?

Reply:
ermmmmmm I feel a little sketchy about overloading focus... I'll add a comment instead :P